pkg/kubelet/kuberuntime/kuberuntime_container_linux.go

```diff
@@ -215,6 +215,13 @@ func (m *kubeGenericRuntimeManager) calculateLinuxResources(cpuRequest, cpuLimit
 		resources.CpuPeriod = cpuPeriod
 	}
 
+	// runc requires cgroupv2 for unified mode
+	if libcontainercgroups.IsCgroup2UnifiedMode() {
+		resources.Unified = map[string]string{
+			// Ask the kernel to kill all processes in the container cgroup in case of OOM.
+			"memory.oom.group": "1",
```

Comment:
Let's add a link to the documentation please. LGTM after that:
https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html